frontend/src/components/Dashboard/CrossingListPage/CrossingListHeader/CrossingListFilter.js

```diff
@@ -20,15 +17,12 @@ class CrossingListStatusCounts extends Component {
       showClosed,
       showCaution,
       showLongterm,
-      params,
     } = this.props;
 
     let openCrossingCount,
       closedCrossingCount,
       cautionCrossingCount,
       longtermCrossingCount;
-    openCrossingCount = closedCrossingCount = cautionCrossingCount = longtermCrossingCount =
```

Comment:
Just to clarify, this will change it so it shows:
Open
instead of:
Open (...)

That seems like a good change to me, just making sure we don't have some reason to use the ... to imply loading or something

Reply:
Yeah, I got rid of the `...`. I don't think loading indicators are super helpful next to the counts.